requirements.txt

```diff
@@ -1,7 +1,19 @@
-Django==1.8.4
-django-extensions
+Django==1.8.5
 ipaddress==1.0.7
 psycopg2==2.6.1
 pygithub3==0.5.1
 requests==2.7.0
 factory-boy==2.5.2
+
```

Comment:
Sadly, this setup is not possible on heroku, because deployments there are "hardwired" to look at <project_root>/requirements.txt. (See note above.)